aqua/emacs-mac-app/Portfile

```
@@ -134,7 +134,12 @@ post-destroot {
 
     # rename the app bundle to avoid a conflict with emacs-app.
     move ${destroot}${applications_dir}/Emacs.app \
-         ${destroot}${applications_dir}/EmacsMac.app
+        ${destroot}${applications_dir}/EmacsMac.app
+
+    # install emacs-module.h header file
+    xinstall -m 755 -d ${destroot}${prefix}/include/emacs-mac
+    xinstall -m 644 ${worksrcpath}/src/emacs-module.h \
```

Very minor style suggestion
```suggestion
    xinstall -m 0755 -d ${destroot}${prefix}/include/emacs-mac
    xinstall -m 0644 ${worksrcpath}/src/emacs-module.h \
```